.github/workflows/codecov.yml

```diff
@@ -14,6 +16,18 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: taiki-e/install-action@cargo-llvm-cov
+      # From: https://github.com/rustls/rustls/blob/3751e24bbc1fef4f8d0bb3711a3be2a3d2a7b793/.github/workflows/build.yml#L59-L71
+      - name: "rustls: Install NASM for aws-lc-rs on Windows"
```

Comment:
I think this first one might not be needed anymore, since rustls no enables `prebuilt-nasm` on aws-lc-rs?